src/main/kotlin/com/expedia/graphql/generator/types/EnumBuilder.kt

```diff
@@ -13,7 +14,7 @@ internal class EnumBuilder(generator: SchemaGenerator) : TypeBuilder(generator)
     internal fun enumType(kClass: KClass<out Enum<*>>): GraphQLEnumType {
         val enumBuilder = GraphQLEnumType.newEnum()
 
-        enumBuilder.name(kClass.getSimpleName())
+        enumBuilder.name(kClass.getGraphQLName() ?: kClass.getSimpleName())
```

Comment:
Can we move the logic behind `getSimpleName` so that we only have to update one location?

Reply:
Yes, you're right, that'd be better. As a pleasant side-effect, it would solve an issue I found earlier today related to building types already under construction: see `kClass.getSimpleName()` in https://github.com/ExpediaDotCom/graphql-kotlin/blob/ceb6b9a162f6956bf5a1c05e1e4e8d00129c9ec3/src/main/kotlin/com/expedia/graphql/generator/state/TypesCache.kt#L77

What's customary in this project: add a new commit or fixup with push-force to the PR?

Reply:
Either is fine, Github will squash all the commits anyway when we merge to master